src/api/rest/actions/orgs.js

```diff
@@ -0,0 +1,7 @@
+import { createActionSet } from 'utils';
+
+export const ORGS_GET_BY_ID = createActionSet('ORGS_GET_BY_ID');
+export const ORGS_GET_REPOS = createActionSet('ORGS_GET_REPOS');
```

Comment:
When I started this PR, I was trying to have the following namespace set up:

```
client.orgs.* //everything related to orgs operation, like
client.orgs.getById(orgId)
client.orgs.getMembers(orgId)

client.users.get() // gets the authenticated user
```

The goal is to namespace the API by entities. It seems to me more clear and well dispatched.

But `connect()` only supports plain functions, so I had to import { getById } from './orgs/';

I was thinking about renaming `orgs.getById()` to `orgsGetById`, `users.get` to `usersGet`, etc, so that it's more clear what is the real function we're calling